ci/ignore_astyle.txt

```diff
@@ -0,0 +1 @@
+components/esp_wifi_remote/include/esp_wifi_types_native.h
```

Comment:
Why is astyle ignored on this header file?

Reply:
good point, I'll add a comment and explain why it's in the ignore list

Reply:
added comment